ledger/sandbox/src/main/scala/com/digitalasset/platform/index/StandaloneIndexServer.scala

```diff
@@ -38,38 +39,21 @@ object StandaloneIndexServer {
       readService: ReadService,
       writeService: WriteService,
       authService: AuthService,
-      loggerFactory: NamedLoggerFactory): StandaloneIndexServer =
+      loggerFactory: NamedLoggerFactory,
+      engine: Engine = engineSharedAmongIndexServers, // allows sharing DAML engine with DAML-on-X participant
```

Comment:
I'm starting to think that this class does a little too much.

Reply:
Is the purpose of this to not have to compile packages twice?

Reply:
@gerolf-da , yes indeed